subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/runtimeshaded/RuntimeShadedJarCreatorTest.groovy

```diff
@@ -302,9 +302,9 @@ org.gradle.api.internal.tasks.CompileServices"""
         then:
         def bytecode = writer.toString()
         !bytecode.contains('LDC "org.apache.ivy.core.settings.XmlSettingsParser"')
-        bytecode.contains('static synthetic Ljava/lang/Class; class$org$gradle$internal$impldep$org$apache$ivy$core$settings$IvySettings')
-        bytecode.contains('GETSTATIC org/gradle/internal/impldep/org/apache/ivy/core/settings/IvySettings.class$org$gradle$internal$impldep$org$apache$ivy$core$settings$IvySettings : Ljava/lang/Class;')
-        bytecode.contains('LDC "org.gradle.internal.impldep.org.apache.ivy.core.settings.IvySettings"')
+        //bytecode.contains('static synthetic Ljava/lang/Class; class$org$gradle$internal$impldep$org$apache$ivy$core$settings$IvySettings') //TODO: this sucks, test shouldn't rely on external code, which changes unexpectedly
```

Comment:
What is the status of this TODO? Do you plan to resolve this as part of the PR?

Reply:
No, I don't intend to resolve it. I don't really understand that area. It's meant as a comment/reminder for whoever will work in that area next to improve that test.